src/UICrafter.Client/Components/LoadMetaDataWeb.razor

```diff
@@ -0,0 +1,18 @@
+@inherits BaseLoadMetaDataList
+
+@{
+    base.BuildRenderTree(__builder);
+}
+
+@code {
+    [Parameter]
+    public EventCallback<long> ClickFunction { get; set; }
+
+    protected override void HandleClick(AppView view)
+    {
+        if (ClickFunction.HasDelegate)
+        {
+            ClickFunction.InvokeAsync(view.Id);
```

Comment:
Use the overload I made TryInvokeAsync HandleClick(AppView view) => ClickFunction.TryInvokeAsync (view.Id);